<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatSet</class>
 <widget class="QWidget" name="FormatSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>票号计数方式设置</string>
   </property>
   <widget class="QRadioButton" name="oneTicketNumRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>每操作票每票号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="oneTicketPageNumRadio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>每操作票页每票号</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="addHeadTailCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>添加页头页尾(#表示票号、$表示页号)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showHeadTailCheck">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作项目中显示页头页尾</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="headEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>页头</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>页尾</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tailEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>176</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>票号格式设置</string>
   </property>
   <widget class="QCheckBox" name="prefixCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>前缀</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="suffixEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="suffixCheck">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>后缀</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fixlengthEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>62</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="fixlenthCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>62</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>固定长度</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>票号设置</string>
   </property>
   <widget class="QCheckBox" name="modifyPreTicketNumCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改预开票号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>22</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>下一预开票号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nextTicketNumEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>81</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>流水号始于</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="modifySerialNumCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>82</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改流水号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beginSerialNumEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="modifyTicketNumCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>52</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改模板、临时票号</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>521</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>操作票输入设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
